client/public/css/stripe.css

```diff
@@ -60,8 +60,6 @@ input {
   width: 20px;
   height: 20px;
   box-shadow: inset 0 0 0 2px;
-  -webkit-transform: translateZ(0);
```

Comment:
why were these deleted?

Reply:
Im not sure, maybe they are no longer needed. This wasn't done manually